src/ape/managers/chain.py

```diff
@@ -438,3 +438,15 @@ def _get_test_provider(self, method_name: str) -> TestProviderAPI:
             )
 
         return provider
+
+    def mine(self, num_blocks: int = 1, timestamp: Optional[int] = None) -> str:
+        if timestamp:
+            self.pending_timestamp = timestamp
+        self.provider.mine(num_blocks)
+        return f'Mining: {num_blocks}, {timestamp}'
```

Comment:
```suggestion
```

Reply:
if me commenting on all of these is annoying, let me know. I just use this as a check list